Analytics-CSharp/Segment/Analytics/Configuration.cs

```diff
@@ -65,8 +69,10 @@ public class Configuration
             ICoroutineExceptionHandler exceptionHandler = null,
             IStorageProvider storageProvider = default)
         {
+            var systeminfo = SystemInfo.get();
+
             this.writeKey = writeKey;
-            this.persistentDataPath = persistentDataPath;
+            this.persistentDataPath = persistentDataPath ?? Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
```

Comment:
would this work on unity on mobile? since it requires to be `Application.persistentDataPath` for unity

Reply:
Yeah, it worked when I tried it and returned the same value that I got from .Net.  I didn't test Unity's Application.persistentDataPath, but from googling it should return the same values.  I'm guessing it's a convenience thing?